client.go

```diff
@@ -35,7 +35,8 @@ import (
 )
 
 const (
-	ClientVersion = internal.ClientVersion
+	// ClientVersion is the semantic versioning compatible client version.
+	ClientVersion = internal.CurrentClientVersion
```

Comment:
Why don't we use `internal.ClientVersion` but use `internal.CurrentClientVersion` which cannot be changed via command line options?

Reply:
Variables cannot be assigned to constants. And we have to use a variable in order to be able to change the version at build time.

Reply:
What is the reason of having `ClientVersion` in client.go that is not used ?

Reply:
Ah ok it is public API to be used by the user. Ok